src/features/bookings/helpers/getLocationLabel.ts

```diff
@@ -9,6 +9,8 @@ export function getLocationLabel(
     return ''
   }
   const { venue } = stock.offer
-  const displayNameVenue = venue.publicName ?? venue.name
+
+  // Do not use ?? as Sonar suggests because if venue.publicName is an empty string we will display an empty string
+  const displayNameVenue = venue.publicName || venue.name
   return displayNameVenue + (venue.city ? `, ${venue.city}` : '')
```

Comment:
Dans ces cas-là j'aime bien avoir un truc du genre 
```suggestion
  const displayNameVenue = venue.publicName || venue.name
  return [displayNameVenue, venue.city].join(', ')
```
c'est un peu plus court et un peu plus lisible.

Reply:
Ta suggestion ça le fait pas si `venue.city` est `null`

Reply:
Ah mince j'ai oublié un morceau pardon, je corrige

Reply:
C'est bon c'est corrigé :D

Reply:
C'est pas forcément plus court du coup :D
Moi perso je suis plus habitué à la façon dont s'est fait actuellement donc pour moi c'est pas forcément plus simple.
@YassinL tu en penses quoi ?

Reply:
la première est plus simple à lire mais j'avoue que j'ai plus vue celle proposer par Donovan donc je ne sais pas trop

Reply:
Je vais update du coup